waiter/src/waiter/util/utils.clj

```diff
@@ -133,18 +133,13 @@
 (defn stringify-elements
   [k v]
   (cond
-    (vector? v) (mapv (partial stringify-elements k) v)
-    (keyword? v) (keyword->str v)
+    (contains? #{true false nil} v) v
+    (map? v) v
+    (number? v) v
+    (sequential? v) (mapv (partial stringify-elements k) v)
```

Comment:
`sequental?` matches vectors, lists, lazy sequences, and persistent queues, but does not match maps or `nil`.
http://clojuredocs.org/clojure.core/sequential_q

The `mapv` will convert any sequential-type into a vector (i.e., array), and ensures all elements of the sequence are also recursively processed. (We weren't converting `PersistentQueue` elements in the old code.)